src/main.ts

```diff
@@ -94,8 +94,11 @@ export async function run(): Promise<void> {
     if (error instanceof Error) {
       core.setFailed(error.message)
     } else {
-      core.setFailed('An unexpected error occurred')
+      core.setFailed(`An unexpected error occurred: ${JSON.stringify(error)}`)
```

Comment:
```suggestion
      core.setFailed(`An unexpected error occurred: ${JSON.stringify(error. null, 2)}`)
```

Small nit, add some new lines so an engineers looking at errors, isnt trying to parse a very long line

Reply:
Sure, good idea